### PR TITLE
Svc 7682

### PR DESCRIPTION
**Pull request by radha-ayla**
IRA #: SVC-7682

# Summary of Changes

pushmeup not including the sound file name

# Documentation link:
# Checklist:
Do you see skip_authorization or skip_permit in the code - No
Self-documenting code - Yes
For validation errors (status 422), error messages defined - Yes
Error messages have corresponding codes under locale/codes - Yes
Documentation complete with working curl examples - Yes
Enough log messages for debuggability - Yes
Security checklist
Were there any changes to how Personally Identifiable Information (PII) is handled, in-transit or at-rest, including in log files, as a result of this change? - YES
If yes, please explain - Obscure potential PII from the logs of external service calls
Logger calls do not contain PII. If a logger call interpolates a serialized object, make sure there’s no PII inside - No
Response do not contain keys - No
Are new packages/gems being added? - No
License Model:
Was the pkg modified to fit our environment? If yes, what is the License model the pkgs authors make it available under? (e.g. GPL, CDDL, CopyLeft, BSD, MIT, etc.) - No new pkg

### CODE REVIEW

File: lib/pushmeup/apnsv3/notification.rb [hunk 1]
```diff
@@ -35,7 +37,7 @@ def body
 
     def to_hash
       aps = {}
-
+      Rails.logger.debug "[Pushmeup::APNSV3::initialize] to_hash #{sound}"
```
Comment (Yipei) [inline lib/pushmeup/apnsv3/notification.rb]: please change the debug message to reflect the correct method  name

**Comment by Yipei** [comment]
How about Badge or Data part mentioned by Fei?

File: lib/pushmeup/apnsv3/notification.rb [hunk 2]
```diff
@@ -35,11 +39,11 @@ def body
 
     def to_hash
       aps = {}
-
       aps.merge!(alert: alert) if alert
       aps.merge!(badge: badge) if badge
       aps.merge!(sound: sound) if sound
       aps.merge!(category: category) if category
+      aps.merge!(other: other) if other
```
Comment (Yipei) [inline lib/pushmeup/apnsv3/notification.rb]: This is how it is added in here https://github.com/AylaNetworks/pushmeup/blob/master/lib/pushmeup/apns/notification.rb

      aps.merge!(self.other) if self.other

Looks like the format will be different

**Comment by wiquan** [comment]
This PR should have incremented the dotRelease in lib/version.rb

**Pull request merged**